dist/api_data.py

```diff
@@ -1430,6 +1430,10 @@ def __ge__(self, other):
 ]),
 
 'WT_SESSION.begin_transaction' : Method([
+    Config('allow_read_timestamp_before_oldest', 'false', r'''
```

Comment:
A five word sentence as a config setting makes it hard to remember the exact order of all the words. What do you think of `read_before_oldest=true`?

Reply:
That does sound better to me, I'll make the change.